lib/src/flagsmith_client.dart

```diff
@@ -88,6 +100,60 @@ class FlagsmithClient {
     return null;
   }
 
+  Future<void> _setupRealtimeUpdates(String realtimeUpdatesBaseUrl) async {
+    final sseUrl = '$realtimeUpdatesBaseUrl$apiKey/stream';
+
+    _sseStream = SSEClient.subscribeToSSE(
+      method: SSERequestType.GET,
+      url: sseUrl,
+      header: {
+        acceptHeader: '$applicationJson , text/event-stream',
+      },
+    );
+
+    _sseStream.listen(
```

Comment:
Just check that we shouldn't be using the optional onError: param here https://api.dart.dev/stable/3.1.0/dart-async/Stream/listen.html

Reply:
Not working unfortunately :(